vault/identity_store.go

```diff
@@ -215,11 +215,6 @@ func mfaPaths(i *IdentityStore) []*framework.Path {
 					Description: `The unique identifier for this MFA method.`,
 					Required:    true,
 				},
-				"entity_id": {
-					Type:        framework.TypeString,
-					Description: "Entity ID on which the generated secret needs to get stored.",
-					Required:    true,
-				},
```

Comment:
This is not related to login enforcement listing, but I verified with Hamid yesterday that this shouldn't be part of this endpoint.